ansible/playbooks/dev/main.yaml

```diff
@@ -0,0 +1,13 @@
+- hosts: all
+
+  tasks:
+  - name: Update apt repos cache
+    become: yes
+    remote_user: ubuntu
```

Comment:
`remote_user` is not required here, I guess — it does not change anything, since you have already defined `ansible_user: ubuntu` in the inventory

Reply:
And I believe you don't really need this task at all if the `docker` role runs successfully without it (for me it did not)